src/indent.c

```diff
@@ -123,14 +123,22 @@ tabstop_padding(colnr_T col, int ts_arg, int *vts)
 
 /*
  * Find the size of the tab that covers a particular column.
+ *
+ * If this is being called as part of a shift operation, col is not the cursor
+ * column but is the column number to the left of the first non-whitespace
+ * character in the line.  If the shift is to the left (left = 1), then return
+ * the size of the tab interval to the left of the column.
```

Comment:
It should also be `TRUE` instead of `1` in this comment
```suggestion
 * character in the line.  If the shift is to the left (left = TRUE), then
 * return the size of the tab interval to the left of the column.
```

Reply:
Yep. Thanks. Done.